### ISSUE TITLE
Can't get example working

### ISSUE THREAD

**Issue by LekeFly** [iantearle/forecast.io-javascript-api#10]
```javascript
forecast ForecastIO {API_KEY: undefined, url: "proxy.php"}
Synchronous XMLHttpRequest on the main thread is deprecated because of its detrimental effects to the end user's experience. For more help, check http://xhr.spec.whatwg.org/.
GET http://domain.dev/forecast.io-javascript-api-master2/proxy.php52.4308,13.2588 404 (Not Found)
Uncaught TypeError: condition.getTemperature is not a function
```

proxy.php returns:
```javascript
Warning: file_get_contents(https://api.forecast.io/forecast/b962d5ee80be5293a234b69fb975629c/https://api.forecast.io/forecast/7f359b3d962d1ae615e1414a52e020fa/52.4308,13.2588): failed to open stream: HTTP request failed! HTTP/1.1 400 Bad Request in forecast.io-javascript-api-master2/proxy.php on line 11
```

i guess this is because of:
```php
$url = $url . $_GET['url'];
```
if i change it to $url = $_GET['url']; i get the json

**Comment by rjbultitude** [comment]
@iantearle I notice that line 47 of forecast.io.js has gone awry in that the url string is not constructed correctly. 
It currently reads:
var requestUrl = this.url + latitude + ',' + longitude;
whereas it should be 
var requestUrl = this.url + '?url=' + latitude + ',' + longitude + '?units=auto';

Are you able to fix and retest?

I would do it on my fork but I'm in the midst of refactoring it so that it can handle multiple location requests and work asynchronously.

**Comment by iantearle** [comment]
fixed by f3eb2db
